### ISSUE TITLE
systemd-udevd: Failed to wait spawned

### ISSUE THREAD

**Issue by kouros17** [systemd/systemd#11341]
I'm using manjaro and gnome desktop (with gdm) and I’m getting the following warning in the journal after the **systemd 240.1-2** until now **240.58-1**

```
systemd-udevd[464]: Failed to wait spawned command '/usr/bin/alsactl restore 1': Input/output error
systemd-udevd[446]: Failed to wait spawned command '/usr/bin/alsactl restore 0': Input/output error
```

I managed to resolve it by following the instructions from [here](https://community.chakralinux.org/t/how-to-solve-usr-sbin-alsactl-restore-0-failed-with-exit-code-99/7047) although I’m not sure if it’s a real fix or a workaround.

**Comment by yuwata** [comment]
Do you mean that the udev rules work with v239 but not with v240? If so, please provide debugging logs of udevd.

**Comment by kouros17** [comment]
How I can provide debugging logs of udevd?
What do you think about [this](https://community.chakralinux.org/t/how-to-solve-usr-sbin-alsactl-restore-0-failed-with-exit-code-99/7047) fix?

**Comment by keszybz** [comment]
> What do you think about this fix?

I'm sorry, but it's not a fix at all. It just disables the functionality.

Please run udev with debugging enabled (put `udev_log=debug` in `/etc/udev/udev.conf`), and paste the logs here.

**Comment by kouros17** [comment]
The issue resolved with **systemd  240.156-1** :+1: 
Thanks a lot @keszybz!
